gitpod-io/website
# Migration to Cypress 11

Signed-off-by: Siddhant Khare <siddhant@gitpod.io>

## Description

This PR Bring Migration of Cypress to latest version ( Cypress 11). This Would resolve all the issues related to e2e tests ( timeout, no-connection issues with cypress 9 & firefox, etc.) -- Would provide a reliable Integration Test Automation.

## Related Issue(s)



Fixes #3089

## How to test

- Open Workspace & Run `npm run test:e2e`

    OR

- :eyes: See Below in GitHub Actions Details

## Release Notes



```release-note
NONE
```
